apps/remix-ide/src/app/tabs/test-tab.js

```diff
@@ -242,13 +264,30 @@ module.exports = class TestTab extends ViewPlugin {
     } else if (result.type === 'testFailure') {
       if (result.hhLogs && result.hhLogs.length) this.printHHLogs(result.hhLogs, result.value)
       if (!result.assertMethod) {
+        let debugBtn = yo``
+        if (result.errMsg.includes('Transaction has been reverted by the EVM')) {
+          const txHash = JSON.parse(result.errMsg.replace('Transaction has been reverted by the EVM:', '')).transactionHash
+          const { web3 } = result
+          debugBtn = yo`<div
```

Comment:
perhaps the debug button should have the same format as of the other actions in the tab (e.g an icon followed by a label).

Reply:
For now, this looks very composed there on the card. @LianaHus also agreed on it as it is now.